ginga/rv/plugins/Pick.py

```diff
@@ -1461,6 +1518,24 @@ def _make_report(self, vip_img, qs):
             rpt_x = x + self.pixel_coords_offset
             rpt_y = y + self.pixel_coords_offset
 
+            # EE at sampling radius
+            try:
+                from scipy.interpolate import interp1d
+                ee_circ_fn = interp1d(
```

Comment:
Theoretically, the fitting functions could also be done inside `self.iqcalc.evaluate_peaks()` above and store as elements in `qs` but I am not sure if it is worth the extra memory... but it would avoid code duplication in plotting.